src/components/smart_objects/src/smart_object.cc

```diff
@@ -723,16 +723,28 @@ void SmartObject::duplicate(const SmartObject& OtherObject) {
 void SmartObject::cleanup_data() {
   switch (m_type) {
     case SmartType_String:
-      delete m_data.str_value;
+      if (m_data.str_value) {
```

Comment:
@AGaliuzov 
Done in commit [`461e69f`](https://github.com/smartdevicelink/sdl_core/pull/660/commits/461e69fc7510ed435e71a6aa0e00242a128221ff)